pages/common/spike.md

```diff
@@ -0,0 +1,19 @@
+# spike
+
+> Spike is a full featured static website generator written in JavaScript.
```

Comment:
Not sure we need to mention `Spike` in the description since we know this page is about it. Perhaps `A fully featured static`..... would read better?